### PR TITLE
Refactor/tx builder

### PR DESCRIPTION
**Pull request by jordanlesich**
## GitHub Issue

(https://github.com/orgs/HausDAO/projects/3/views/3)

## Changes

- See Issue description 

## Packages Added

Brief list of any packages added, and if folks need to rerun `yarn install` to run locally

## Checks

Before making your PR, please check the following:

- [x] Critical lint errors are resolved
- [x] App runs locally
- [x] App builds locally (run the build command for *any impacted package* and check for any errors before the PR)

### CODE REVIEW

File: tsconfig.base.json
Comment (jordanlesich) [inline tsconfig.base.json:18]: I'm curious why it generated w/ @daohaus-monorepo instead of @daohaus

Comment (skuhlmann) [reply tsconfig.base.json:18]: and i see it the other way here:

https://github.com/HausDAO/daohaus-monorepo/pull/304/files#diff-520b36df452487d613b8ea62dfc687f081a918373e6e085bebf388136f0d6e0aR2

feel like we need to fix this before merging. can manually change it.

**Review by skuhlmann — CHANGES_REQUESTED** [review]

**Review by jonathanprozzi — APPROVED** [review]
Approved!

**Comment by jordanlesich** [comment]
Hey @skuhlmann, the requested changes we blocking the merge. I did make the changes you suggested, but I did it locally.

**Pull request merged**
